### ISSUE TITLE
Feature Request: Pinout text label for transistor object

### ISSUE THREAD

**Issue by iiimonfiiire** [bancika/diy-layout-creator#511]
It would be great to have a way to automatically label the transistor pinouts. Right now, I create a transistor object and if I want to have the pinout values display on the object in my layout, then I have to either: 
1) specify it in the "Name" or "Value" fields (and select "Name" or "Value" from the object Display menu). However, this messes up with my BOM. Or:
2) group the transistor object with a separate text label object, which usually has to be oriented accordingly with the transistor (see attached screenshot). However, if I need to change the orientation of the transistor on my layout, then I also need to ensure that the pinout label is also rotated along with the transistor. 

**So in short, the feature request is:**
1) Enable "Pinout" as an option in the Display menu for the Transistor object
2) Ensure that the pinout is rotated accordingly whenever the Transistor object is rotated

![image](https://user-images.githubusercontent.com/2866557/78545179-05c1a300-7804-11ea-88c1-908fe7cf579e.png)

**Comment by bancika** [comment]
added Display -> Pinout for transistor packages.

![image](https://user-images.githubusercontent.com/11462281/79080914-206ead00-7d19-11ea-920c-f2f126cf0fd7.png)
